pkg/scheduler/framework/interface.go

```diff
@@ -898,6 +902,12 @@ func (ni *NominatingInfo) Mode() NominatingMode {
 	return ni.NominatingMode
 }
 
+// PodActivator abstracts operations in the scheduling queue.
+type PodActivator interface {
+	// Activate moves the given pods to activeQ iff they're in unschedulablePods or backoffQ.
```

Comment:
```suggestion
	// Activate moves the given pods to activeQ if they're in unschedulablePods or backoffQ.
```

Reply:
Ah, ignore me if you mean `if and only if` 😂

Reply:
"if and only if", not a typo.

Reply:
Yeah

Reply:
Me 3 years ago did the same actually, lol 😅 
https://github.com/kubernetes/kubernetes/pull/104542